### PR TITLE
Support DockerRegistry CR in the `image-import` command

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- use the DockerRegistry CR to import image
- if DR does not exist, try to import image to the old (serverless based) registry
- improve the success message after importing

**Related issue(s)**

- https://github.com/kyma-project/cli/issues/2070#issue-2289027892

### CODE REVIEW

File: internal/registry/config.go [hunk 1]
```diff
@@ -4,47 +4,87 @@ import (
...
+	SecretName string
+	SecretData *SecretData
+	PodMeta    *RegistryPodMeta
+}
+
+func GetConfig(ctx context.Context, client kube.Client) (*RegistryConfig, error) {
+	var secretConfig *SecretData
+	var secretName string
+
+	dockerRegistry, err := getServedDockerRegistry(ctx, client.Dynamic())
```
Comment (anoipm) [inline internal/registry/config.go:31]: IMO this part of code is hard to read. Whole method is long.

File: internal/registry/config.go [hunk 2]
```diff
@@ -114,3 +154,24 @@ func isPodReady(pod corev1.Pod) bool {
...
+	}
+
+	for _, item := range list.Items {
+		var dockerRegistry DockerRegistry
+		err = kube.FromUnstructured(&item, &dockerRegistry)
+		if err != nil {
+			return nil, err
+		}
+
+		if dockerRegistry.Status.Served == "True" {
```
Comment (anoipm) [inline internal/registry/config.go:171]: Why you look on `Served`? IMO better could be `State == "Ready"`.

File: internal/registry/types.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package registry
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	DockerRegistryGVR = schema.GroupVersionResource{
```
Comment (anoipm) [inline internal/registry/types.go:9]: Now we have file `internal/btp/operator/gvr.go`. Isn't it good place for this definition?

Comment (pPrecel) [reply internal/registry/types.go:9]: internal/btp package is related to the `btp` platform. And all sub-directories should be have the same destiny. Of course I can move these lines into the `internal/registry/gvr.go` file, but I think `types.go` file is generic enough

File: internal/registry/config.go (hunk 2 above)
Comment (pPrecel) [reply internal/registry/config.go]: done

File: internal/registry/config.go (hunk 1 above)
Comment (pPrecel) [reply internal/registry/config.go]: done

File: internal/registry/config.go (hunk 2 above)
Comment (pPrecel) [reply internal/registry/config.go]: good point, thanks

**Pull request merged**